CHANGES.md

```diff
@@ -25,6 +29,8 @@
 * Fix warning due to changing default `dtype` when creating an empty
   `pandas.Series`.
 * Enable `GridSearch` for more than two sensitive features values.
+* Change `ExponentiatedGradient` signature by renaming argument `T` to
+  `max_iterations`, `eta_mul` to `learning_rate`, and by adding `run_lp_step`.
 * Add new disparity constraints for reductions methods as moments in
```

Comment:
remember to update the names here according to corrections

Reply:
... and it's also in the wrong release because it didn't make 0.4.6 :-)